multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/zeistmanga/ZeistMangaDto.kt

```diff
@@ -0,0 +1,75 @@
+package eu.kanade.tachiyomi.multisrc.zeistmanga
+
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.jsoup.Jsoup
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+@Serializable
+data class ZeistMangaDto(
+    @SerialName("feed") val feed: ZeistMangaFeedDto? = null,
+)
+
+@Serializable
+data class ZeistMangaFeedDto(
+    @SerialName("entry") val entry: List<ZeistMangaEntryDto>? = emptyList()
+)
+
+@Serializable
+data class ZeistMangaEntryDto(
+    @SerialName("title") val title: ZeistMangaEntryTitleDto? = null,
+    @SerialName("published") val published: ZeistMangaEntryPublishedDto? = null,
+    @SerialName("link") val url: List<ZeistMangaEntryLink>? = emptyList(),
+    @SerialName("content") val content: ZeistMangaEntryContentDto? = null
+) {
+    fun toSManga(baseurl: String): SManga = SManga.create().apply {
+        title = this@ZeistMangaEntryDto.title!!.t
+        url = getChapterLink(this@ZeistMangaEntryDto.url!!).substringAfter(baseurl)
+        thumbnail_url = getThumbnail(this@ZeistMangaEntryDto.content!!)
+    }
+
+    fun toSChapter(baseurl: String): SChapter = SChapter.create().apply {
+        name = this@ZeistMangaEntryDto.title!!.t
+        chapter_number = this@ZeistMangaEntryDto.title.t.trim().substringAfterLast(" ").toFloat()
```

Comment:
This will fall if the title doesn't contain a valid number. You don't necessarily need to set this as the app will try to parse the chapter name and get the chapter number automatically. If you fell you really need to set it, do like this:

```suggestion
        chapter_number = this@ZeistMangaEntryDto.title.t.trim().substringAfterLast(" ").toFloatOrNull() ?: -1f
```